mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/components/overview/index.tsx

```diff
@@ -39,15 +39,16 @@ function getEmailDelta(type: 'opened' | 'clicked'): number | undefined {
   const current = getEmailPercentage(type, 'current');
   const previous = getEmailPercentage(type, 'previous');
   if (current === undefined || previous === undefined) {
-    return undefined;
+    return 0;
   }
 
   if (previous === 0) {
     return 0;
   }
 
   const newValue = current > previous ? current - previous : previous - current;
-  return (newValue / previous) * 100;
+  const delta = (newValue / previous) * 100;
+  return current > previous ? delta : delta * -1;
```

Comment:
I guess the original `current > previous ? current - previous : previous - current` is not needed and then it's simply:

```tsx
const newValue = current - previous;
return (newValue / previous) * 100;
```